feature/community/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="communityui_cardbutton_ammenities">Amenities</string>
+    <string name="communityui_cardbutton_events">Events</string>
+    <string name="communityui_cardbutton_communityWall">Community Wall</string>
+    <string name="communityui_cardbutton_doItYourself">Do It Yourself</string>
```

Comment:
Please follow the camel case structure: ``communityUi_cardButton_doItYourself

Reply:
Fixing